### DISCUSSION TITLE
toolbar custom extension button 排序問題

### DISCUSSION THREAD

**Discussion by a0926019412** [dotnetcore/BootstrapBlazor#2996] [Q&A] [answered]
### Which class is this unit test associated with?

請問, table toolbar default buttons & custom extension buttons 是否可以做排序呢?

**Comment by bb-auto** [comment]
@a0926019412 Thank you for submitting. We will give feedback later.

**Comment by a0926019412** [comment]
有找到 TableToolbarBeforeTemplate
thanks~

**Comment by ArgoZhang** [comment, marked as answer]
顺序就是 

`TableToolbarBeforeTemplate` > `AddButton` > `EditButton` > `DeleteButton` > `TableToolbarTemplate`
